src/oph/ehoks/ehoks_app.clj

```diff
@@ -77,22 +79,27 @@
   "Ready-to-call app with oppija app-routes and session middleware."
   (common-api/create-app oppija-handler/app-routes (session-store/db-store)))
 
+(def palaute-app
+  "Ready-to-call app with palaute app-routes and session middleware"
+  (common-api/create-app palaute-handler/app-routes (session-store/db-store)))
+
 (def app-by-name
   {"ehoks-virkailija" #'virkailija-app
    "ehoks-oppija" #'oppija-app
-   "both" #'both-app})
```

Comment:
tän "both":n voisi ehkä pitää taaksepäin yhteensopivuuden vuoksi.  Myös README.md viittaa vielä siihen.

Reply:
voi kyl olla ettei kukaan oo käyttänyt sitä :)

Reply:
Palautin tämän takaisin muotoon "both"

Reply:
juu, siis saa se "all":kin siellä olla, ei siitä mitään haittaa ole.  Mutta millä tavalla vain.